.travis.yml

```diff
@@ -60,6 +65,25 @@ matrix:
           - haskell-platform
           - rpm
           - fakeroot
+  - language: scala
+    env: SUBPROJECT=p2p-test-network
+    scala: 2.12.4
+    sbt_args: -no-colors
+    install:
+      - ./scripts/install_secp.sh
+      - ./scripts/install_sodium.sh
+      - ./scripts/install_bnfc.sh
+      - ./scripts/p2p-test-network.sh
+    addons:
+      apt:
+        sources:
+          - sourceline: 'deb https://dl.bintray.com/sbt/debian /'
```

Comment:
What's bintray?

Reply:
It's used for sbt install
https://www.scala-sbt.org/download.html - adds it to sources.
This helps support versions other than in the travis build